src/library/searchquery.cpp

```diff
@@ -157,8 +169,16 @@ bool TextFilterNode::match(const TrackPointer& pTrack) const {
 
 QString TextFilterNode::toSql() const {
     FieldEscaper escaper(m_database);
-    QString escapedArgument = escaper.escapeString(kSqlLikeMatchAll + m_argument + kSqlLikeMatchAll);
-
+    QString argument = m_argument;
+    if (argument.size() > 0) {
+        if (argument[argument.size() - 1].isSpace()) {
+            // LIKE eats a tailing space. This can be avoided by adding a '_'
```

Comment:
"_t **r** ailing_" is the correct term as far as I remember. This typo also appears in the name of a test.

Reply:
.. and I thought is should be "tailing" ... fixed.